algorithms/kernel/dtrees/forest/classification/df_classification_train_kernel.h

```diff
@@ -50,6 +50,8 @@ class ClassificationTrainBatchKernel : public daal::algorithms::Kernel
 public:
     services::Status compute(HostAppIface * pHostApp, const NumericTable * x, const NumericTable * y, decision_forest::classification::Model & m,
                              Result & res, const decision_forest::classification::training::interface1::Parameter & par);
+    services::Status compute(HostAppIface * pHostApp, const NumericTable * x, const NumericTable * y, decision_forest::classification::Model & m,
```

Comment:
Why do you have 3 compute? You need copy/cast to terrible interface (to interface3). For support BC new parameters you set in such a way that the logic remains the same. I am sure this is the best way.

Reply:
Fixed